packages/react-dev-utils/BuildProgressPlugin.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const path = require('path');
+
+let ProgressPlugin;
+if (__dirname.indexOf(path.join('packages', 'react-dev-utils')) !== -1) {
+  ProgressPlugin = require('../react-scripts/node_modules/webpack').ProgressPlugin;
```

Comment:
Webpack doesn't exist in any parent directory during development, only a sister directory (which node's module resolution doesn't traverse).
For development in the monorepo to work, we need a hack. I chose to do it this way, though we could accomplish it however.

tl;dr without it: `Error: Cannot find module 'webpack'`